packages/react-ai/src/components/AIConversation/views/Controls/InputControl.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+
+import { withBaseElementProps } from '@aws-amplify/ui-react-core/elements';
+
+import { AIConversationElements } from '../../context/elements';
+import { AttachFileControl } from './AttachFileControl';
+
+const { Button, Icon, TextArea, View } = AIConversationElements;
+
+const INPUT_BLOCK = 'ai-input';
+
+const sendIconProps = () => ({
+  children: (
+    <svg>
+      <g clipPath="url(#clip0_387_1426)">
+        <path
+          d="M14.3199 7.98214L1.66669 1.23486L3.97674 7.98214M14.3199 7.98214L1.66669 14.5689L3.97674 7.98214M14.3199 7.98214H3.97674"
+          strokeWidth="1.5"
+          strokeLinecap="square"
+          strokeLinejoin="round"
+        />
+      </g>
+      <defs>
+        <clipPath id="clip0_387_1426">
+          <rect width="100%" height="100%" />
+        </clipPath>
+      </defs>
+    </svg>
```

Comment:
- where did this icon come from?
- seems to be more complex than our other icons that just contain a `path` element?
- extraneous wrapping `svg`

Reply:
From the Raven figma. happy to change it it there's somewhere else i should be pulling from